azkaban-common/src/main/java/azkaban/AzkabanCommonModule.java

```diff
@@ -76,10 +71,4 @@ protected void configure() {
       throw new StorageException(e);
     }
   }
-
-  @Inject
-  public @Provides
-  LocalStorage createLocalStorage(AzkabanCommonModuleConfig config) {
```

Comment:
No. Because I already have an `@inject` annotation in the class. Guice gets confused if there are no annotations and it has multiple constructors like we saw the other day.

Bindings are mandatory for interfaces where Guice doesn't know what implementation to use.